src/client/providers/jediHelpers.ts

```diff
@@ -49,8 +49,8 @@ export function extractSignatureAndDocumentation(definition: proxy.IAutoComplete
 }
 
 export function highlightCode(docstring: string): string {
-    // Add blank line
-    docstring = EOL + EOL + docstring + EOL + EOL;
```

Comment:
These `EOL` should be kept.
Because some blocks are ended with 'blank line' or `EOF` (end of file), I added two `EOL` (making a blank line) before and after the docstring to make it easier to match with regex.

To fix the unit tests, I think `trim()` the docstring at the end of this function makes more sense.